scripts/publish_pr.py

```
@@ -0,0 +1,389 @@
+import sys
+from pathlib import Path
+from typing import Optional
+
+import requests
+
+
+def get_repo_info() -> tuple[str, str]:
+    """Extract owner and repo from git remote origin."""
+    try:
+        result = subprocess.run(
+            ["git", "config", "--get", "remote.origin.url"],
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)